packages/editor/src/components/preferences-modal/index.js

```diff
@@ -314,6 +326,7 @@ export default function EditorPreferencesModal( { extraSections = {} } ) {
 			setIsListViewOpened,
 			setPreference,
 			isLargeViewport,
+			starterPatterns,
```

Comment:
We probably don't need to break the memo when there's a new starter pattern? I think we only need to check whether there are `starterPatterns` or not. We can create a `hasStarterPatterns` boolean for this.

Reply:
Good call! I'll update.